### ISSUE TITLE
Font patch error for SourceHanCodeJP

### ISSUE THREAD

**Issue by MartinP7r** [ryanoasis/nerd-fonts#641]
Hello and thank you for providing this tool!

I'm getting basically the same error as #412 I think.

```bash
// lots of unicode errors above
No glyph with unicode U+05c60 in font
No glyph with unicode U+06753 in font
No glyph with unicode U+06eba in font
No glyph with unicode U+07c3e in font
Traceback (most recent call last):
  File "/root/nerd-fonts/font-patcher", line 908, in <module>
    main()
  File "/root/nerd-fonts/font-patcher", line 902, in main
    patcher = font_patcher()
  File "/root/nerd-fonts/font-patcher", line 55, in __init__
    self.setup_font_names()
  File "/root/nerd-fonts/font-patcher", line 267, in setup_font_names
    fullname = self.sourceFont.fullname + verboseAdditionalFontNameSuffix
TypeError: unsupported operand type(s) for +: 'NoneType' and 'unicode'
```

#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_ SourceHanCodeJP
- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_ iterm
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_ docker

**Comment by ryanoasis** [comment]
Are you still having this problem with the latest version of the patcher or the docker registry?

#439 should've fixed the issue if it was indeed the same as #412

**Comment by MartinP7r** [comment]
Thank you @ryanoasis for having a look at this!  
I honestly haven't tried since then.  
I'm going to check it out towards the end of the month when I'm back from vacation.  
Feel free to close this and I'll re-open if necessary.

**Comment by ryanoasis** [comment]
Thanks. Sounds like a fair plan. Enjoy!

Closing for now

**Comment by Finii** [comment]
Works on my machine when you give the extension.

![image](https://user-images.githubusercontent.com/16012374/146230224-8abe5614-fd30-4dec-b76f-4f5b4b3a36b7.png)

There is clearly something wrong with the extension extraction stuff whatever ;)

**Comment by Finii** [comment]
Edit: Although the line seems to be, as suspected, before #439.

Maybe we should change the issue-template to include information on font-patcher version.

The line number in the error message is clearly Jimmy Haas' commit (i.e. pre #439)

![image](https://user-images.githubusercontent.com/16012374/146231501-0c703a2c-7a1a-4959-8826-171ebafdcf42.png)

**Comment by ryanoasis** [comment]
fixed via #720 which is now merged. Thanks. Re-open if issue persists.
